src/Toolbar/ToolbarItem.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ElementType, FC, HTMLAttributes, MouseEvent } from 'react';
+import classNames from 'classnames';
+import { Icon } from '../Icon/Icon';
+
+type HTMLElementWithDisabled<T> = Partial<T> & { disabled?: boolean };
+
+export interface ToolbarItemProps
+  extends HTMLElementWithDisabled<HTMLAttributes<HTMLElement>> {
```

Comment:
```suggestion
  extends HTMLAttributes<HTMLElement> {
```
With the change above